Scuti/Scripts/UI/Account/BannerWidget.cs

```diff
@@ -138,19 +138,30 @@ private async void ShowOfferDetails(bool isVideo)
         var offer = await ScutiNetClient.Instance.Offer.GetOfferByID(_banner.Id.ToString());
         if (offer != null)
         {
-            var panelModel = Mappers.GetOfferDetailsPresenterModel(offer);
-            if (panelModel != null)
+            if (!ScutiUtils.HasLink(offer))
             {
-                UIManager.OfferDetails.SetData(panelModel);
-                UIManager.OfferDetails.SetIsVideo(isVideo);
-                UIManager.Open(UIManager.OfferDetails);
-              
+                var panelModel = Mappers.GetOfferDetailsPresenterModel(offer);
+                if (panelModel != null)
+                {
+                    UIManager.OfferDetails.SetData(panelModel);
+                    UIManager.OfferDetails.SetIsVideo(isVideo);
+                    UIManager.Open(UIManager.OfferDetails);
+                } 
             }
```

Comment:
Probably need to track that the link was clicked here.  

Maybe:

                        ScutiAPI.EngagementWithProductMetric(0, 1, offer.ID);